token/services/ttxdb/db/badger/init.go

```diff
@@ -25,9 +27,9 @@ type Opts struct {
 type Driver struct {
 }
 
-func (d Driver) Open(sp view2.ServiceProvider, name string) (driver.AuditDB, error) {
+func (d Driver) Open(sp view2.ServiceProvider, name string) (driver.DB, error) {
 	opts := &Opts{}
-	err := view2.GetConfigService(sp).UnmarshalKey("token.auditor.auditdb.persistence.opts", opts)
+	err := view2.GetConfigService(sp).UnmarshalKey("token.ttxdb.persistence.opts", opts)
```

Comment:
since you touch this here, can we refactor this key into a const somewhere here in the code?

Reply:
there is already some const for another key